tfx_addons/copy_example_gen/component_test.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+Tests for tfx_addons.copy_example_gen.component.
+"""
+from unittest import mock
+
+import tensorflow as tf
+
+from tfx_addons.copy_example_gen import component
+
+
+class TestCopyExampleGen(tf.test.TestCase):
+  """Test module for CopyExampleGen."""
+  def setUp(self):
+    self.input_json_str = """
+    {
+      "label1": "fakeuri",
+      "label2": "fakeuri2",
+    }
+    """
+
+  def test_empty_input(self) -> None:
+    empty_input_json_str = ""
+    with self.assertRaises(ValueError):
+      component.create_input_dictionary(input_json_str=empty_input_json_str)
+
+  def test_non_dictionary_input(self) -> None:
+    non_dictionary_input = "'a', 'b', 'c'"
+
+    with self.assertRaises(ValueError):
+      component.create_input_dictionary(input_json_str=non_dictionary_input)
+
+  def test_empty_dictionary(self) -> None:
+    empty_input_json_str = "{}"
+    # TODO(zyang7): Decide if an empty dictionary should be allowed or if an
+    # exception should be thrown.
```

Comment:
Looking for feedback on what the expected behavior should be, thanks!

Reply:
An empty dictionary entails no split-names or Tfrecord URIs, therefore, an exception should be thrown.